nixos/modules/programs/ssh.nix

```diff
@@ -62,6 +62,66 @@ in
         '';
       };
 
+      pubkeyAcceptedKeyTypes = mkOption {
+        type = types.listOf types.str;
+        # Allow DSA keys for now. (These were deprecated in OpenSSH 7.0.)
+        default = [
```

Comment:
I think it may be better to not list all the types, but stay with the `+ssh-dss` instead. This way we get the default recommended options from the ssh package and don't have to remember to update this list when the recommendations change. Does that seem sensible?

Reply:
To me it does. I don't see any advantage in having to list the default types if you just want to add some.

Reply:
I also strongly prefer the OpenSSL defaults. I doubt we will keep track of this tightly.

I would also not want to introduce something that upstream has been deprecated as that could mean we can not take it out again. See the issue about deprecating the dsa support.

Reply:
I would argue for removing some of the doubtful `nistp` algorithms but we probably can not be sure nobody relies on them.. That is a very sad story.